tests/wpt/mozilla/tests/mozilla/DOMParser-parseFromString.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
```

Comment:
How would you feel about making this a crash test (I think you just need the `-crash.html` suffix) and adding it to the WPT? I believe that there is more leeway with adding crash tests there that don't necessarily exercise bugs in other engines.

Reply:
I think it would run the risk of being removed as a duplicate of existing tests, since it's literally just making one basic API call that's already exercised in other files. Once we're running all of domparser/ with the async parser then we shouldn't need this one anymore.